pkg/lifecycle/action.go

```diff
@@ -16,10 +16,10 @@
 
 package lifecycle
 
-// Action - policy action.
+// PolicyAction - policy action.
```

Comment:
We need to update `pkg/iam/policy/action.go` to allow users to set lifecycle if necessary using IAM policies under `supportedActions` @vadmeste

Reply:
> We need to update `pkg/iam/policy/action.go` to allow users to set lifecycle if necessary using IAM policies under `supportedActions` @vadmeste

Can we move the definition of these permissions in pkg/iam ? I don't like seeing iam depending on lifecycle

Reply:
Ideally yes.